features/step_definitions/form_steps.rb

```diff
@@ -6,6 +6,6 @@
   end
 end
 
-When /^I click on the label "(.*?)"$/ do |label_text|
+When /^I click on the label "([^"]+)"$/ do |label_text|
```

Comment:
```suggestion
When "I click on the label {string}" do |label_text|
```

Reply:
Just a question, `string` is not the name of the var, but the type?